sentry-android-replay/src/main/java/io/sentry/android/replay/ScreenshotRecorder.kt

```diff
@@ -163,16 +160,15 @@ internal class ScreenshotRecorder(
                                 }
                                 return@traverse true
                             }
-                        }
 
-                        val screenshot = scaledBitmap.copy(ARGB_8888, false)
-                        screenshotRecorderCallback?.onScreenshotRecorded(screenshot)
-                        lastScreenshot?.recycle()
-                        lastScreenshot = screenshot
-                        contentChanged.set(false)
+                            val screenshot = bitmap.copy(ARGB_8888, false)
+                            screenshotRecorderCallback?.onScreenshotRecorded(screenshot)
+                            lastScreenshot?.recycle()
+                            lastScreenshot = screenshot
+                            contentChanged.set(false)
 
-                        scaledBitmap.recycle()
-                        bitmap.recycle()
+                            bitmap.recycle()
```

Comment:
I guess at some point we could check if recycling the bitmap is really required.

Reply:
yep, was planning to profile memory and see how it behaves - but a bit later, after the beta probably